KthLargestElement.py

```diff
@@ -0,0 +1,29 @@
+class Solution(object):
+    def findKthLargest(self, nums, k):
+        """
+        :type nums: List[int]
+        :type k: int
+        :rtype: int
+        """
+        return self.quickSelect(nums, k)
```

Comment:
Do you really need to define `quickSelect`?
Guess you can just recursively call `findKthLargest` to do the same thing?

Reply:
Comments has been addressed.